spec/services/catalog/copy_portfolio_item_spec.rb

```diff
@@ -0,0 +1,45 @@
+describe Catalog::CopyPortfolioItem do
+  let(:tenant) { create(:tenant) }
+  let(:portfolio) { create(:portfolio, :tenant_id => tenant.id) }
+  let(:portfolio2) { create(:portfolio, :tenant_id => tenant.id) }
+  let(:portfolio_item) { create(:portfolio_item, :portfolio_id => portfolio.id, :tenant_id => 1) }
```

Comment:
@lindgrenj6 why :tenant_id is 1 and not tenant.id?

Reply:
That was a mistype on my part, fixing it.